internal/client/tanzukubernetescluster/tanzukubernetescluster_resource.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package tanzukubernetesclusterclient
+
+import (
+	"net/url"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/client/transport"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/helper"
+	tkcmodels "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/tanzukubernetescluster"
+)
+
+const (
+	apiVersionAndGroup      = "/v1alpha1/managementclusters"
```

Comment:
The `/` before v1aplpha1 is needed?

Reply:
Not needed, but it should be no harm as it is trimmed while url is formed. https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/main/internal/client/transport/methods.go#L41